prime-router/metadata/hl7_mapping/Senders/TrustedIntermediaryLabOrder.yml

```diff
@@ -0,0 +1,2 @@
+extends: ../OML_O21/OML_O21-base
```

Comment:
This folder set-up is temporary for now. Sender transforms are still being worked on and once that is done the place where this file exists will change, but its fine here for now.

Reply:
Speaking of "sender transforms" which are still a work in progress, I see our `prime-router/settings/staging/0149-etor.yml` which mentions the `OML_O21-base` YAML file (line `58`) for a _receiver_.  Should that be updated to the new `TrustedIntermediaryLabOrder.yml` file that is in this PR?  Line `58` is under receiving though, and this `TrustedIntermediaryLabOrder.yml` file is all about sending.  So maybe we shouldn't update line `58` to the new file.  Thoughts?

Reply:
I'd say let's wait and see how all this shakes out once everything is restructured, then we can think about how all this is organized. We can investigate issues as they come up and see if we need to update these files.

Reply:
Sounds like a plan.